app/src/main/res/values/strings.xml

```diff
@@ -173,6 +173,7 @@
     <string name="activity_login_log_in">Log in</string>
     <string name="activity_login_sign_up">Sign up for SUSI</string>
     <string name="activity_login_skip">Skip</string>
+    <string name="activity_signup_skip">Skip</string>
```

Comment:
What's the difference between this and the string above?

Reply:
Sir, I added the string only because those two strings belong to two different pages. So, I thought it would be better to seperate to define it sepeartely. If asked i will remove it.

Reply:
No, strings don't depend on their page, only on their content